### DISCUSSION TITLE
Is it just me or is "Process Image" not sending meta data to "From Image" again ?

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#1209] [General]
I am just getting the first word of the positive prompt, and the first word of the negative prompt sent over. This worked yesterday, but has changed since I --upgrade today (May 29, 2023 5f1fd7bd).

Thanks !

**Comment by vladmandic** [comment]
Ahhh, I might have broken something, I was refactoring thar code today...

**Reply by swamp-cabbage** [reply]
No complaints here - Thanks Bro !

**Reply by vladmandic** [reply]
don't have time to fully test it, should be fixed now.

**Reply by swamp-cabbage** [reply]
FIXED (un gran aplauso)

**Reply by vladmandic** [reply]
btw, the whole concept how metadata is sent between controls is probably my least favorite part of the legacy code.
only worse part was metadata handling in general which i've completely replaced, but now i have to deal with this until i replace it as well (eventually).
